packages/app/components/claim/claim-button.tsx

```diff
@@ -336,7 +298,7 @@ export const ClaimButton = ({
         {edition?.apple_music_track_url ? (
           <Button
             {...buttonProps}
-            onPress={() => handleCollectPress("appleMusic")}
+            onPress={handleClaimNFT}
```

Comment:
> handleCollectPress("appleMusic")

this button explicitly passes appleMusic. i.e. the user has pressed apple music so it triggers apple music flow because a pre save drop can have apple music URL as well as spotify URL. I think we'll have to handle it in handleClaimNFT

Reply:
Oh yes, that makes sense. I forgot about that.

Reply:
fixed!